frontend/language/src/nb.json

```diff
@@ -1608,6 +1608,8 @@
   "ux_editor.options_text_label": "Ledetekst",
   "ux_editor.page": "Side",
   "ux_editor.page_config_pdf_abort_converting_page_to_pdf": "Avbryt å gjøre om siden til PDF",
+  "ux_editor.page_config_pdf_card_heading": "Siden skal være en PDF",
+  "ux_editor.page_config_pdf_card_text": "Når siden settes som en PDF, blir den ikke lenger en del av sidestrukturen, men bare en PDF som vises ( der den vises (oppsummering, utskrift eller som vedlegg?)). Den vil da plasseres nederst blant siden markert som en PDF.",
```

Comment:
This text will be updated before merge. Waiting for a better alternative 😄

Reply:
@Ildest, @Annikenkbrathen og @wrt95, når en side er angitt i egenskapen `pdfLayoutName`, vil den ikke lenger være en del av selve skjemautfyllingen. Når appen genererer PDF-en etter innsending, er det denne siden som danner grunnlaget for PDF-genereringen. Sluttbrukeren vil da få tilgang til PDF-en med innholdet fra denne spesifikke siden. Dersom brukeren derimot ikke angir en eksplisitt side for PDF, vil alle sidene bli brukt som grunnlag, og de vil legges etter hverandre i PDF-en som genereres.


En annen ting, som går litt utenfor det opprinnelige spørsmålet, er at når en side legges til i `pdfLayoutName`, støtter den kun følgende komponenter i siden som benyttes som PDF:

- **Summary**
- **Summary2** (gir mulighet til å lage PDF med alle komponenter). _Denne er underutvikling fra vår side nå._
- **Group**
- **InstanceInformation**
- **Header**
- **Paragraph**
- **Image**
- **Panel**

Har vi sjekket at vi har implementert denne begrensningen i Studio? Hvis ikke, burde vi nok e på det. 😊

Reply:
Text is now updated 😄